README.md

```diff
@@ -244,48 +238,38 @@ The omniauth-google-oauth2 gem supports this mode of operation out of the box.
 
 ```javascript
 // Basic hybrid auth example following the pattern at:
-// https://developers.google.com/api-client-library/javascript/features/authentication#Authexample
-jQuery(function() {
-  return $.ajax({
-    url: 'https://apis.google.com/js/client:plus.js?onload=gpAsyncInit',
-    dataType: 'script',
-    cache: true
-  });
-});
-
-window.gpAsyncInit = function() {
-  gapi.auth.authorize({
-    immediate: true,
-    response_type: 'code',
-    cookie_policy: 'single_host_origin',
-    client_id: 'YOUR_CLIENT_ID',
-    scope: 'email profile'
-  }, function(response) {
-    return;
-  });
-  $('.googleplus-login').click(function(e) {
-    e.preventDefault();
-    gapi.auth.authorize({
-      immediate: false,
-      response_type: 'code',
-      cookie_policy: 'single_host_origin',
-      client_id: 'YOUR_CLIENT_ID',
-      scope: 'email profile'
-    }, function(response) {
-      if (response && !response.error) {
-        // google authentication succeed, now post data to server.
-        jQuery.ajax({type: 'POST', url: '/auth/google_oauth2/callback', data: response,
-          success: function(data) {
-            // response from server
-          }
-        });
-      } else {
-        // google authentication failed
-      }
+// https://developers.google.com/identity/sign-in/web/reference
+
+<script src="https://apis.google.com/js/platform.js?onload=init" async defer></script>
```

Comment:
The newer JS library appears to expect the client ID to be provided in the HEAD via meta tags:

```
<meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com">
```

If you don't, it'll raise an error saying as such. If you do specify the client ID via a meta tag, you can't use the `gapi.auth2.authorize` end-point. Instead I suspect you need to get the auth instance and call the `signIn` API:

```
  ... click(function(){
    var auth2 = gapi.auth2.getAuthInstance();
    auth2.signIn().then(function (request) {
      // Post the request to the Omniauth google endpoint
    });
  });
```